spark/v3.1/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSparkSqlExtensionsParser.scala

```diff
@@ -255,18 +253,20 @@ case object IcebergParseErrorListener extends BaseErrorListener {
 }
 
 /**
- * Copied from Apache Spark
- * A [[ParseException]] is an [[AnalysisException]] that is thrown during the parse process. It
- * contains fields and an extended error message that make reporting and diagnosing errors easier.
+ * Copied from Apache Spark A [[ParseException]] is an [[AnalysisException]] that is thrown during
```

Comment:
I think the line break here is intended shouldn't be joined with the next line.

Reply:
Will take a look

Reply:
Fixed